### DISCUSSION TITLE
Convective rainfall and snowfall

### DISCUSSION THREAD

**Discussion by chloe-eagle** [MetOffice/simulation-systems#130] [UM]
Hello, I am running a regional nesting suite. Please could you tell me how I can output convective rainfall (m01s05i205) and convective snowfall (m01s05i206) to a file? Currently it wont output. Is there something I need to do? Or what is the reason it wont work and what is the alternative please?

**Comment by Adrian-Lock** [comment]
Regional UM configurations typically have the convection scheme turned off, which is why you then can't access these diagnostics.  If you're only wanting to know what the total precip is then the large-scale diagnostics (which the RNS outputs in the basic verstash) is all you need.  If you want to discriminate in the regional model between rainfall generated by convective (Cu, Cb) as opposed to "large-scale" (frontal, drizzling Sc) processes then you need to do some diagnostic postprocessing.  People have looked into this but I can't immediately put my hand on a suitable reference!

**Reply by chloe-eagle** [reply]
Thank you. Please do you know how I could turn the convective scheme on in the RNS?

**Reply by iboutle** [reply]
The first question is probably why you'd want to do that? There are many different options for how to do it, but none of them are particularly good, hence why it isn't used as standard.
